package/aws/certificate/definition.yaml

```diff
@@ -0,0 +1,153 @@
+                      log.
+                    enum:
+                    - ENABLED
+                    - DISABLED
+                    type: string
+                  domainName:
+                    description: Fully qualified domain name (FQDN),that to secure
+                      with an ACM certificate.
+                    type: string
+                  domainValidationOptions:
+                    description: The domain name that you want ACM to use to send
+                      you emails so that you can validate domain ownership.
+                    items:
+                      description: DomainValidationOption validate domain ownership.
+                      properties:
+                        domainName:
+                          description: Additinal Fully qualified domain name (FQDN),that
+                            to secure with an ACM certificate.
+                          type: string
+                        validationDomain:
+                          description: Method to validate certificate
+                          type: string
+                      required:
+                      - domainName
+                      - validationDomain
+                      type: object
+                    type: array
+                  region:
+                    description: Region is the region you'd like your Certificate
+                      to be created in.
+                    type: string
+                  renewCertificate:
+                    description: Flag to renew the certificate
+                    type: boolean
+                  subjectAlternativeNames:
+                    description: Subject Alternative Name extension of the ACM certificate.
+                    items:
+                      type: string
+                    type: array
+                  tags:
+                    description: One or more resource tags to associate with the certificate.
+                    items:
+                      description: Tag represents user-provided metadata that can
+                        be associated
+                      properties:
+                        key:
+                          description: The key name that can be used to look up or
+                            retrieve the associated value.
+                          type: string
+                        value:
+                          description: The value associated with this tag.
+                          type: string
+                      required:
+                      - key
+                      - value
+                      type: object
+                    type: array
+                  validationMethod:
+                    description: Method to validate certificate.
+                    enum:
+                    - DNS
+                    - EMAIL
+                    type: string
+                  deletionPolicy: 

```

Comment:
duplicate from line 48. Why this is happening again? do we need to fix something in the readme or some bad example?

Reply:
No I think it's because I added it to the top to be consistent with the others, bearing in mind this PR was started before the README, and I overlooked removing it from the bottom